x-pack/plugins/monitoring/public/directives/main/index.js

```diff
@@ -3,13 +3,57 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import {
+  EuiSelect,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
 import { get } from 'lodash';
 import { uiModules } from 'ui/modules';
 import template from './index.html';
 import { shortenPipelineHash } from '../../../common/formatting';
 import 'ui/directives/kbn_href';
 
+const setOptions = (controller) => {
+  if (!controller.pipelineVersions || !controller.pipelineVersions.length || !controller.pipelineDropdownElement) {
+    return;
+  }
+
+  render(
+    <EuiFlexGroup>
+      <EuiFlexItem grow={false}>
+        <EuiTitle style={{ maxWidth: 400, lineHeight: '40px', overflow: 'hidden', whiteSpace: 'nowrap' }}>
+          <h2>{controller.pipelineId}</h2>
+        </EuiTitle>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiSelect
+          value={controller.pipelineHash}
```

Comment:
On IE11, I'm seeing a warning here:
```
checkPropTypes.js:20 Warning: Failed prop type: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.
    in select (created by EuiValidatableControl)
    in EuiValidatableControl (created by EuiSelect)
```

Reply:
I couldn't reproduce this for some reason. The `EuiSelect` does have a `value` and an `onChange` event which are not null at any point